source/upgrade-guide/index.rst

```diff
@@ -1,23 +1,29 @@
-.. Copyright (C) 2022 Wazuh, Inc.
+.. Copyright (C) 2015, Wazuh, Inc.
 
 .. meta::
-  :description: Find out more about the process of upgrading the Wazuh server, Open Distro for Elasticsearch, Elastic Stack, and Wazuh agents in this section.
+   :description: Find out more about the process of upgrading the Wazuh central components, Open Distro for Elasticsearch, Elastic Stack, and Wazuh agents in this section.
   
-.. _upgrade_guide:
-
 Upgrade guide
 =============
 
-This guide describes the upgrade process of the Wazuh server, Open Distro for Elasticsearch, Elastic Stack, and the Wazuh agents. The upgrading from a legacy version section is for those users who need to upgrade the Wazuh installation from a version before 3.0.
+This guide gives instructions to upgrade your Wazuh installation.
+
+-  :doc:`Upgrade Wazuh central components <upgrading-central-components>`
+-  :doc:`Upgrade Wazuh and Open Distro for Elasticsearch <elasticsearch-kibana-filebeat/upgrading-open-distro>`
+-  :doc:`Upgrade Wazuh and Elastic Stack basic license <elasticsearch-kibana-filebeat/upgrading-elastic-stack>`
+-  :doc:`Upgrade Wazuh agents <upgrading-agent>`
```

Comment:
I would differentiate between 4 sections:
- Upgrade Wazuh server, Wazuh dashboard and Wazuh indexer.
- Upgrade Wazuh server and Open Distro for Elasticsearch.
- Upgrade Wazuh server and Elastic Stack basic license.
- Upgrade Wazuh agents.